system/logaccess/Azure/LogAnalytics/CurlClient/AzureLogAnalyticsCurlClient.cpp

```diff
@@ -1081,6 +1081,151 @@ bool AzureLogAnalyticsCurlClient::processSearchJsonResp(LogQueryResultDetails &
     return true;
 }
 
+bool AzureLogAnalyticsCurlClient::healthReport(StringBuffer & report, LogAccessHealthReportOptions options)
+{
+    try
+    {
+        report.appendf("\"ConnectionInfo\": { \"TargetALAWorkspaceID\": \"%s\" ", m_logAnalyticsWorkspaceID.str());
+        report.appendf(", \"TargetALATenantID\": \"%s\"", m_aadTenantID.str());
+        report.appendf(", \"TargetALAClientID\": \"%s\"", m_aadClientID.str());
+        report.appendf(", \"TargetALASecret\": \"%sempty\"", m_aadClientSecret.length()==0 ? "" : "not ");
+        report.appendf(", \"TargetsContainerLogV2\": \"%s\"", targetIsContainerLogV2 ? "true" : "false");
+        report.appendf(", \"ComponentsQueryJoins\": \"%sabled\"", m_disableComponentNameJoins ? "dis" : "en");
+        report.appendf(", \"BlobModeUnstructuredLogData\": \"%sabled\"", m_blobMode ? "en" : "dis");
+        report.append( "}"); //close conninfo
+
+        report.append(", \"ConfigurationInfo\": { ");
+
+        if (m_pluginCfg)
+        {
+            StringBuffer configJSON;
+            toJSON(m_pluginCfg, configJSON, 0);
+            report.appendf("\"ConfigurationTree\": %s", configJSON.str()); //json encode
```

Comment:
Same question here as in [ElasticStackLogAccess.cpp](https://github.com/hpcc-systems/HPCC-Platform/pull/19307/files#diff-c2677d642c90c214a1fba89d682b912ed919c7a29d8372d6211145972257b2e5) about JSON vs JSON in a string which needs encoding.